_includes/v19.2/sql/movr-statements.md

```diff
@@ -0,0 +1,5 @@
+The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application and dataset, see [MovR: A Global Ride-sharing App](movr.html).
+
+To follow along with the example statements, do one of the following:
+
+{% include {{page.version.version}}/sql/movr-start.md %}
```

Comment:
Is it necessary to have the double-layer of includes here?  Maybe I'm not understanding the whole structure of this.
General comment on the include strategy: I wonder also if we could get away with having one `movr-statements.md` include rather than having one each for with/without data.  Maybe there's a way to write it up as a choice, like "in case A, do this; in case B, do this"?

Reply:
The intro sentence really only applies to SQL statement examples (e.g. insert.md), but the movr-start.md instructions apply to all examples where we use the MovR dataset (e.g. learn-cockroachdb-sql.md). If we could use conditional includes, that would be amazing. Candidly, I don't know how to do that... I'll follow up offline about this.

Reply:
SGTM